pygmt/src/timestamp.py

```diff
@@ -0,0 +1,106 @@
+"""
+timestamp - Plot the GMT timestamp logo.
+"""
+from packaging.version import Version
+from pygmt.clib import Session
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import build_arg_string, is_nonstr_iter
+
+__doctest_skip__ = ["timestamp"]
+
+
+def timestamp(
+    self,
+    text=None,
+    label=None,
+    justification="BL",
+    offset=("-54p", "-54p"),
+    font="Helvetica",
+    timefmt="%Y %b %d %H:%M:%S",
+):
+    r"""
+    Plot the GMT timestamp logo.
+
+    Parameters
+    ----------
+    text : None or str
+        If ``None``, the current UNIX time stamp is shown in the GMT timestamp
+        logo. Setting this parameter to replace the UNIX time stamp with a
+        custom text string instead. The text must be less than 64 characters.
+        *Requires GMT>=6.5.0*.
```

Comment:
I'm still unsure what's the best way to document features that require new GMT versions.

Reply:
Hm. Is there a similar situation for the Julia wrapper, which can serve as an orientation?